src/arrayMethodSort.js

```diff
@@ -3,9 +3,58 @@
 /**
  * Implement method Sort
  */
+
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
```

Comment:
create a default function compareAsStrings outside applyCustomSort

```suggestion
  [].__proto__.sort2 = function(compareFunction = compareAsStrings) {
```